hack/test-cmd.sh

```diff
@@ -1087,13 +1087,11 @@ __EOF__
   kube::test::get_object_assert deployment "{{range.items}}{{$deployment_image_field}}:{{end}}" "${IMAGE_NGINX}:${IMAGE_NGINX}:"
   kube::test::if_has_string "${output_message}" "Object 'Kind' is missing"
   ## Pause the deployment
-  output_message=$(! kubectl rollout pause -f hack/testdata/recursive/deployment --recursive 2>&1 "${kube_flags[@]}")
```

Comment:
Would you change the title comment of this test too (L1077)? These tests are about "--recursive" and the "rollout" commands, but not about "Rollback a deployment". It's somewhat misleading. Changing each sub-title would help too (maybe add "recursively" to the end of each sub-title, and mention that we expect error message "Object 'Kind' is missing").